googletest/include/gtest/gtest.h

```diff
@@ -2116,9 +2116,11 @@ GTEST_API_ AssertionResult DoubleLE(const char* expr1, const char* expr2,
 // of the dummy variable name, thus allowing multiple SCOPED_TRACE()s
 // to appear in the same block - as long as they are on different
 // lines.
-#define SCOPED_TRACE(message) \
-  ::testing::internal::ScopedTrace GTEST_CONCAT_TOKEN_(gtest_trace_, __LINE__)(\
-    __FILE__, __LINE__, ::testing::Message() << (message))
+#define SCOPED_TRACE_(message, file, line) \
```

Comment:
`SCOPED_TRACE2`? `SCOPED_TRACE_EX`? I'm calling it `MY_SCOPED_TRACE`, but it feels bad rewriting macros that use internal APIs.